Parsers/ASimAuthentication/Parsers/ASimAuthenticationVMwareCarbonBlackCloud.yaml

```diff
@@ -0,0 +1,73 @@
+Parser:
+  Title: ASIM Authentication parser for VMware Carbon Black Cloud
+  Version: '0.1.0'
+  LastUpdated: Oct 09 2023
+Product:
+  Name: VMware Carbon Black Cloud
+Normalization:
+  Schema: Authentication
+  Version: '0.1.3'
+References:
+- Title: ASIM Authentication Schema
+  Link: https://aka.ms/ASimAuthenticationDoc
+- Title: ASIM
+  Link: https:/aka.ms/AboutASIM
+- Title: VMware Carbon Black Cloud documentation
+  Link: 
+    https://developer.carbonblack.com/reference/carbon-black-cloud/data-forwarder/schema/latest/endpoint.event-1.0.0/
+    https://developer.carbonblack.com/reference/cb-threathunter/latest/event-search-fields/
+Description: |
+  This ASIM parser supports normalizing VMware Carbon Black Cloud logs to the ASIM Authentication normalized schema. VMware Carbon Black Cloud events are captured through VMware Carbon Black Cloud data connector which ingests Carbon Black Audit, Notification and Event data into Microsoft Sentinel through the REST API.
+ParserName: ASimAuthenticationVMwareCarbonBlackCloud
+EquivalentBuiltInParser: _ASim_Authentication_VMwareCarbonBlackCloud
+ParserParams:
+  - Name: disabled
+    Type: bool
+    Default: false
+ParserQuery: |
+  let parser = (disabled: bool=false) {
+      CarbonBlackAuditLogs_CL
+      | where not(disabled)
+      | where description_s has_any ("logged in", "login") and description_s !has "connector"
+      | extend
+          EventStartTime = unixtime_milliseconds_todatetime(eventTime_d),
+          EventResult = iff(description_s has "successfully", "Success", "Failure"),
+          AdditionalFields = bag_pack("flagged", flagged_b),
+          EventSeverity = iff(flagged_b == true, "Low", "Informational")
+      | extend
+          EventCount = int(1),
+          EventProduct = "Carbon Black Cloud",
+          EventSchema = "Authentication",
+          EventSchemaVersion = "0.1.3",
+          EventVendor = "VMware",
+          EventType = "Logon",
+          EventResultDetails = iff(EventResult == "Failure", "Incorrect password", ""),
+          EventOriginalResultDetails = iff(EventResult == "Failure", tostring(split(description_s, ';')[1]), "")
+      | project-rename
+          EventMessage = description_s,
+          EventOriginalUid = eventId_g,
+          TargetUsername = loginName_s,
+          SrcIpAddr = clientIp_s,
+          EventUid=_ItemId,
+          EventOwner = orgName_s
```

Comment:
this was mapped with ActorScope in Audit. In this authentication this could be mapped with TargetUserScope
![image](https://github.com/Azure/Azure-Sentinel/assets/97222872/01799bd1-7946-4708-bee2-9033d5b0b713)

Reply:
@jayeshprajapaticrest @niralishah-crest will take care.